packages/flutter_hooks/lib/src/listenable.dart

```diff
@@ -128,3 +128,15 @@ class _UseValueNotifierHookState<T>
   @override
   String get debugLabel => 'useValueNotifier';
 }
+
+/// Subscribes to a [Listenable] and registers `listener` to the passed `listenable`.
+///
+/// See also:
+///   * [useListenable]
+void useOnListenableChange(void Function() listener, Listenable listenable) {
+  useEffect(() {
```

Comment:
Use the class syntax here

All hooks in the flutter_hooks package use the class syntax, for the same of hot-reload